target/linux/bcm27xx/bcm2712/target.mk

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (C) 2023 OpenWrt.org
+#
+
+ARCH:=aarch64
+SUBTARGET:=bcm2712
+BOARDNAME:=BCM2712 boards (64 bit)
+CPU_TYPE:=cortex-a72
```

Comment:
Shouldn't this be `CPU_TYPE:=cortex-a76`?

Reply:
I set cortex-a72 because there is not a package feed for cortex-a76, changed now.